### ISSUE TITLE
Merge compiling to Lambda(undefined, undefined)

### ISSUE THREAD

**Issue by gsears** [fauna-brecht/fauna-schema-migrate#10]
Hey all,

First of all just want to say this looks like an excellent tool with lots of promise! 

Currently when using `Merge` in some of my queries, I'm getting the migration query output as `Lambda(undefined, undefined)` in a UDF. 

For example, the following UDF definition:

```js
import faunadb from 'faunadb';

const q = faunadb.query;
const { Query, Lambda, CreateFunction, Var, Let, Merge, Select } = q;

const ReturnWithId = doc => {
  return Let(
    { document: doc },
    Merge(Select('data', Var('document')), {
      id: Select(['ref', 'id'], Var('document')),
    })
  );
};

export const ReturnWithIDUDF = CreateFunction({
  name: 'return_with_id',
  body: Query(Lambda(['doc'], ReturnWithId(Var('doc')))),
});

export default ReturnWithIDUDF;
```

Compiles to:
```
CreateFunction({
  name: "return_with_id",
  body: Query(Lambda(["doc"], Let([{
    document: Var("doc")
  }], Lambda(undefined, undefined))))
}
```

Am I missing something or is this a bug?

Keep up the great work!

**Comment by fauna-brecht** [comment]
Definitely a bug, probably in the representation conversation of strings to FQL in the driver or vice versa, will take a look.

**Comment by fauna-brecht** [comment]
Determined the issue, talking with engineering to take a look.

**Comment by fauna-brecht** [comment]
Just to keep you posted, the fix will be released somewhere this or next week in the JS driver.

**Comment by fauna-brecht** [comment]
Should be fixed when pairing it with the 4.1.2 fauna driver.
